medusa/classes.py

```diff
@@ -91,6 +91,9 @@ def __init__(self, episodes):
         # Result type like: nzb, nzbdata, torrent
         self.resultType = u''
 
+        # Store the parse result, as it might be usefull of other information later on.
```

Comment:
"Store the parse result, as it might be ~~usefull~~ useful ~~of~~ for other information later on"